Tests/CarthageKitTests/ProjectSpec.swift

```diff
@@ -424,3 +508,59 @@ extension ProjectEvent {
 		return false
 	}
 }
+
+
+// swiftlint:enable identifier_name
+private struct DB {
```

Comment:
This DB struct is copied 1:1 from `ResolverSpec`. Should it rather be extracted into a separate file?

Reply:
Yes, please extract it and the `github1`, `github2`, etc. 👍